lib/utils/errorHelper.ts

```diff
@@ -0,0 +1,5 @@
+export const checkError = (error: unknown) => {
```

Comment:
To make it more clear what this is checking for would some other name be more descriptive? For example something like `isAssignableToError`?

Reply:
Yeah good point - I might just leave a comment as to why it exists also.